portfolio/src/main/webapp/index.html

```diff
@@ -20,24 +20,27 @@ <h1>Maxwell's Portfolio</h1>
       <p>Click here to get a random greeting from around the world!:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-      
-      <p>Click here to say hello!:</p>
-      <button onclick="fetchMessage()">Fetch Message:</button>
-      <div id="fetch-container"></div>
 
       <form action="/data" method="POST">
         <label for="comment">Enter a Comment:</label>
         <input type="text" id = "comment" name = "comment">
         <br/><br/>
```

Comment:
Using line breaks for spacing isn't the best style.
You don't get control of the amount of space, and if you want to change it, you end up having to update this everywhere you have a similar objective. If this spacing is something you want to do for all submit buttons, then it would make sense to make this part of a "submit-button" CSS class style. If it is a more general use of white space between form input elements, you may wish instead to use `<div>` with a "form-input-space" or "form-input-vspace" class which will set the height of the spacing.

The only advantage you might see from this is that it should scale with font size. But there are other ways to accomplish that by setting the height style property with the unit "em". See https://www.w3schools.com/css/css_units.asp for more details on units.

Reply:
Whoops, forgot I used these. I'll remove the line breaks for now and update the CSS over time.